package.json

```diff
@@ -72,12 +72,14 @@
   "files": [],
   "name": "typescript-sdk",
   "dependencies": {
+    "@opentelemetry/core": "^1.0.1",
+    "@opentelemetry/semantic-conventions": "^1.0.1",
```

Comment:
nit: Seems like we _usually_ put 3rd in `devDependencies` let's keep it consistent.
I think `arg` and `uuid` are used in scripts but even those can be put in dev.